lib/Serialization/Deserialization.cpp

```diff
@@ -4555,6 +4504,93 @@ void ModuleFile::loadAllMembers(Decl *container, uint64_t contextData) {
   }
 }
 
+static Decl* handleErrorAndFudgeMissingMember     (llvm::Error &&,  ASTContext &,  Decl*         container);
```

Comment:
Change order of functions rather than pre-declaring statics. Also avoid vertical alignment; run clang-format.